src/style.css

```diff
@@ -1,3 +1,108 @@
-body {
+html {
+  --color: #00acdc;
+}
+
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+header {
+  width: 100%;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  background-color: #fff;
   margin: 0;
+  font-family: Roboto, sans-serif;
```

Comment:
Remember to use fallback fonts in the `font-family` property to ensure text is displayed with a default font in case the primary font fails to load.